### PR TITLE
Add AST - Assist corp.

### PR DESCRIPTION
**Pull request by ygonno**
Opt Out:

Below is the full description:

  {
    "code": "AST",
    "contact": {
      "address": "4-16-7 Nishi-Kohjiya Ohta-ku Tokyo, Japan",
      "email": "itou@assist-corp.com",
      "name": "Takemasa Ito"
    },
    "description": "Assist corp.",
    "url": "https://www.assist-corp.tv/"
  },

### CODE REVIEW

**Comment by ygonno** [comment]
Did I miss something? I don't understand what is causing such validation failures.

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -403,6 +403,11 @@
     "code": "ASC",
     "description": "ASCENT MEDIA"
   },
+  {
+    "code": "AST",
+    "description": "Assist corp."
```
Comment (palemieux) [inline src/main/data/facilities.json:408]: Comma `,` missing

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by ygonno** [comment]
argh.. my eyes.. thx!

**Comment by palemieux** [comment]
@japierce1028 For your review

**Comment by palemieux** [comment]
@ygonno Can you merge in the latest HEAD of `master` branch?

**Comment by ygonno** [comment]
Done?

**Review by palemieux — APPROVED** [review]

**Comment by palemieux** [comment]
@ygonno It looks good technically. @japierce1028 should review for content.

**Review by japierce1028 — APPROVED** [review]
I assume this came directly and not through the google form submission. In one comment it says "Opt OUT" so the final entry only shows the URL, so seems right to me.

**Comment by japierce1028** [comment]
Wait, when I go to "squash and merge" it shows all the entries. Clearly I don't know what I'm doing. It should ONLY have the name, description and URL -- assuming it is an Opt OUT entry.

**Comment by palemieux** [comment]
> Wait, when I go to "squash and merge" it shows all the entries. Clearly I don't know what I'm doing. It should ONLY have the name, description and URL -- assuming it is an Opt OUT entry.

@japierce1028 You should edit the commit message to remove all irrelevant information.

image

**Comment by japierce1028** [comment]
When I go to ygonno:patch-1 all seems right (only the correct info). So I went ahead and confirmed the merger.

**Pull request merged**